test/test_01_02.py

```diff
@@ -0,0 +1,25 @@
+from typing import Sequence
+
+import pytest
+
+from hm_1.tasks.task_01_02 import check_fibonacci
+
+
+@pytest.mark.parametrize(
+    "value",
+    [[0, 1], [0, 1, 1, 2], [0, 1, 1], [0, 1, 1, 2, 3, 5, 8], [0, 1, 1, 2, 3, 5, 8, 13]],
```

Comment:
Do we have any other special values that should be tested?

Reply:
In my opinion no, because a list size of 1 we don't consider, I don't see another special cases